Core/Frameworks/Baikal/Model/Config.php

```diff
@@ -205,7 +199,7 @@ function persist() {
 
         $sLines = implode("\n", $aLines);
         $sSandboxedCode = str_replace(["<?php", "<?", "?>"], "", $sLines);
-        $sRand = (string)rand();
+        $sRand = (string) rand();
```

Comment:
There are a few places where it adds the space - seems like a reasonable "fix" and not too many lines of diff.